TheHive/thehive/create_alert.py

```diff
@@ -94,17 +94,19 @@ def run(self, arguments: dict[str, Any]):
                             )
                         )
 
+        alert_type = f"{arg_alert['alert_type']['category']}/{arg_alert['alert_type']['value']}"
+        link = f"https://app.sekoia.io/operations/alerts/{arg_alert['short_id']}"
```

Comment:
Will not work with other regions than FRA1

Reply:
Indeed, can we keep it like that for now,
How this type of issue is managed in other module ?

Reply:
Yes we can keep it this way for now. 

I think we will need to address the issue with changes in our API so the action can know in which region they are running.